src/biome/text/configuration.py

```diff
@@ -644,4 +654,297 @@ class PredictionConfiguration:
 
     add_tokens: bool = False
     add_attributions: bool = False
-    attributions_kwargs: Dict = dataclasses.field(default_factory=dict)
+    attributions_kwargs: Dict = field(default_factory=dict)
+
+
+@dataclass
+class LightningTrainerConfiguration:
```

Comment:
My main question would be how are we going to keep up with changes and additions to the `pytorch_lightning.trainer`.  Do not know the specifics but from the outside it seems they evolve really fast, but maybe is something we can cope with, what's your opinion @dcfidalgo ?

Reply:
@dvsrepo Yes, they move fast ... but i think it is not super critical for us to require the latest version. Not as critical as AllenNLP, for example, since they define our pytorch version to work with, which is important for a smooth experience in Colab.

So i think it's feasible to bump up the pl version in our requirements from time to time and update those parameters. That said, i also thought about only exposing the more frequently used parameters in the `TrainerConfiguration` and put the rest in a `lightning_trainer_kwargs`. Maybe this would make this configuration class a bit handier and would make the update to new pl versions a tiny bit easier. But for this i would like to gain a bit more experience with the arguments first.

Reply:
Fully agree, all good on my side, ignore my comment